concepticondata/conceptlists/Dawson-1887-731.tsv

```diff
@@ -157,7 +157,7 @@ Dawson-1887-731-155	155	Mat	195	MAT	Dwellings	4	29
 Dawson-1887-731-156	156	Bed	1663	BED	Dwellings	4	29
 Dawson-1887-731-157	157	Floor	1632	FLOOR	Dwellings	4	29
 Dawson-1887-731-158	158	Ceiling			Dwellings	4	29
-Dawson-1887-731-159	159	Wall	933	WALL (OF HOUSE)	Dwellings	4	29
+Dawson-1887-731-159	159	Wall	3830	WALL	Dwellings	4	29
```

Comment:
Could this be indeed WALL (OF HOUSE) as opposed to the ceiling above?

Reply:
In this instance in can indeed be both (just going by the lexemes), so the more general mapping is okay I'd say.